src/test/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImplTest.java

```diff
@@ -203,19 +205,44 @@ void whenSavingAudioRequestIsSuccessful() {
         requestDetails.setEndTime(OffsetDateTime.parse(OFFSET_T_12_00_00_Z));
         requestDetails.setRequestType(DOWNLOAD);
 
-        when(mockHearingRepository.getReferenceById(hearingId)).thenReturn(mockHearingEntity);
+        when(mockHearingService.getHearingById(hearingId)).thenReturn(mockHearingEntity);
         when(mockMediaRequestRepository.saveAndFlush(any(MediaRequestEntity.class))).thenReturn(mockMediaRequestEntity);
         when(mockUserAccountRepository.getReferenceById(TEST_REQUESTER)).thenReturn(mockUserAccountEntity);
         doNothing().when(auditApi).record(any(), any(), any());
         var request = mediaRequestService.saveAudioRequest(requestDetails);
 
         assertEquals(request.getId(), mockMediaRequestEntity.getId());
-        verify(mockHearingRepository).getReferenceById(hearingId);
+        verify(mockHearingService).getHearingById(hearingId);
         verify(mockMediaRequestRepository).saveAndFlush(any(MediaRequestEntity.class));
         verify(mockUserAccountRepository).getReferenceById(TEST_REQUESTER);
         verify(auditApi).record(REQUEST_AUDIO, mockUserAccountEntity, mockCourtCaseEntity);
     }
 
+    @Test
+    void saveAudioRequest_whenHearingNotFound_shouldThrowException() {
```

Comment:
shall we follow the same naming style for naming tests that are used in this test class? 
e.g.
`whenSavingAudioRequestIsSuccessful()`